src/thermo/PDSS.cpp

```diff
@@ -184,7 +185,14 @@ void PDSS::setState_TP(doublereal temp, doublereal pres)
 
 void PDSS::setState_TR(doublereal temp, doublereal rho)
 {
-    throw NotImplementedError("PDSS::setState_TR");
+    warn_deprecated("PDSS::setState_TR",
```

Comment:
If this is not implemented anyways, could it just be replaced without warning?

Reply:
It's not implemented by the base class, but is overloaded by specializations.